src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/AbstractCaseView.java

```diff
@@ -46,4 +47,11 @@ public void setTabs(CaseViewTab[] tabs) {
     public void setMetadataFields(List<CaseViewField> metadataFields) {
         this.metadataFields = metadataFields;
     }
+
+    public void addMetadataFields(List<CaseViewField> metadataFields) {
```

Comment:
May be with single item rather than list? 
void addMetadataFields(CaseViewField metadataField)

Reply:
Could probably do what Sateesh suggested, but instead of replacing this method, add a new one to support adding a singe CaseViewField to the list, as the current one you have could also be useful

Reply:
👍